### DISCUSSION TITLE
Something about the corner on the screen

### DISCUSSION THREAD

**Discussion by x-leehe** [end-4/dots-hyprland#661] [Q&A] [answered]
Hey guys I'm using Arch Linux with hyprland(with Illogical Impulse configuration) now and I've using it for more than roughly 3 months. It's working well, smooth and efficient. However there's still something disrupted me: How can I turn off the rounded corner on the screen? I've been searching in the configuration files for some time but still have no answer.😥

But still a nice job. Despite some fcitx5 issues(that's hyprland's problem), it is really a great job, ~~which amazed my friends.~~

![Screenshot_2024-06-28_17.49.40.png](https://github.com/end-4/dots-hyprland/assets/78580595/41151dcd-48ba-4544-bd6d-f82946b51d5b)

**Comment by end-4** [comment, marked as answer]
appearance.fakeScreenRounding = 0

**Reply by PopCat19** [reply]
Am I supposed to write it in `ags/user_options.js`? 

I know it's a variable in `/ags/modules/.configuration/user_options.js`, but I don't want it to change every update.

Adding it to `hyprland.conf` doesn't do anything. Adding it under `const userConfigOptions = {` in `user_options.js` causes AGS to crash.

Sorry for inexperience.

**Reply by end-4** [reply]
> Am I supposed to write it in `ags/user_options.js`?

yes. options in here override defaults in `ags/modules/.configuration/user_options.js`

**Comment by lifeashansen** [comment]
This has always worked for me without any problems.
Set visiblity to false in 

.config/ags/modules/screencorners/main.js
